### ISSUE TITLE
Merge new features with flow-for-vscode?

### ISSUE THREAD

**Issue by orta** [gcazaciuc/vscode-flow-ide#3]
Hi, I'd love to get anything your working on inside flow-for-vscode so that we can all work together on an extension.

Is there anything I can do to help migrate anything flow-for-vscode in?

**Comment by kurtextrem** [comment]
Friendly push :) @gcazaciuc @orta

**Comment by gcazaciuc** [comment]
Sorry for dropping the ball on this. The problem with the official extension is that i consider it way too bloated in terms of code.
Initially i wanted to contribute directly to the official extension in order to add the features i wanted and debug the code.
I gave up pretty quickly because the code is huge for what it does and goes in through many layers which are very hard to follow.
Besides this the vscode flow-types don't keep up with the VSCode API and I definitely consider them extremely helpful, so i believe writing the extension in the best tool for the given context(in this case typescript) is a plus.
On top of that there was a lot of Nuclide specific code which made things more confusing.

For the time being I consider the extension that i maintain far easier to do maintenance work on it and I would prefer continue supporting it for the above reasons.

**Comment by gcazaciuc** [comment]
Closing this for now since it is not actionable at the moment.
